src/index.js

```diff
@@ -0,0 +1,4 @@
+import './css/style.css'
```

Comment:
Codacy has a fix for the issue: [Replace `'./css/style.cs` with `Tabs·from·'./classes/Tab`](https://app.codacy.com/gh/BeAPI/accessible-components/pullRequest?prid=8000264)
```suggestion
import Tabs from './classes/Tabs'
```